src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -197,6 +197,27 @@ def test_pyttsx(pyi_builder):
         """)
 
 
+@importorskip('pyttsx3')
+def test_pyttsx3(pyi_builder):
+    import os
+
+    SOURCE = """
+        import os
+        import pyttsx3
+        engine = pyttsx3.init()
+    """
+    # The above is enough to cause a failure without the hook. It's a lot more
+    # reassuring if it includes the block below too.
+    # But on Windows CI this blocks indefinitely so exclude it in that case.
+
+    if not (os.environ.get("CI", False) and os.name == "nt"):
+        SOURCE += """
+        engine.say("I do not like green eggs and ham.")
+        engine.runAndWait()
+        """
```

Comment:
I think SOURCE might look nicer if it contained the conditional block with boolean condition being injected via format(), instead of being appended. But it's probably just a matter of preference...